main/admin.py

```diff
@@ -0,0 +1,5 @@
+"""Admin panel code is here."""
```

Comment:
Please do not write useless descriptions stating obvious things like "the second line in this module starts with 'f'". Try taking point of view of a stranger looking at the module his first time and trying to figure out what's going on here.
Write a high-level description of what is the purpose of this module.